inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/gru_sequence.cpp

```diff
@@ -30,10 +34,11 @@ namespace {
 
     INSTANTIATE_TEST_CASE_P(smoke_GRUSequenceCommonZeroClip, GRUSequenceTest,
                             ::testing::Combine(
+                                    ::testing::ValuesIn(mode),
                                     ::testing::ValuesIn(seq_lengths_zero_clip),
                                     ::testing::ValuesIn(batch),
                                     ::testing::ValuesIn(hidden_size),
-                                    ::testing::ValuesIn(input_size),
+                                    // ::testing::ValuesIn(input_size), // hardcoded to 10 due to Combine supports up to 10 args
```

Comment:
In general, you can use nested `Combine`s to workaround such gtest limitation

Reply:
Could you provide some usage examples in code?

Reply:
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convolution.cpp

Reply:
So I also need to change GRU Cell tests, I will do it for both GRU Sequence/Cell in the next PR (ticket 43220)